### PR TITLE
#16730 timezone enrichment documentation

### LINKED ISSUE #16730: Add timezone fix to documentation (fixed for CO, as reference)
## Problem statement

We are no longer able to use the setting convertDateTimeToReceiverLocalTimezone under the receiver translation settings. To support this setting we have developed an enrichment schema that allows us to convert all datetimes in a FHIR bundle to a specific timezone.
The documentation for migrating and onboarding receivers needs to be update to take this into account.

## What you need to know

Documentation that needs to be updated:
- https://github.com/CDCgov/prime-reportstream/blob/main/prime-router/docs/onboarding-users/migrating-receivers.md
- https://github.com/CDCgov/prime-reportstream/blob/main/prime-router/docs/onboarding-users/receivers.md

The documentation needs to mention how to use the new enrichment schema and how to create a new one if necessary.

## Acceptance criteria

- [ ] Documentation is updated with new process on using enrichment schemas to set a receiver's timezone

### PR DESCRIPTION
**Pull request by the-andrew**
Update docs to mentions changes for time zone conversion translation to use enrichment schema

**If you are suggesting a fix for a currently exploitable issue, please disclose the issue to the prime-reportstream team directly outside of GitHub instead of filing a PR, so we may immediately patch the affected systems before a disclosure. See [SECURITY.md/Reporting a Vulnerability](https://github.com/CDCgov/prime-reportstream/blob/main/SECURITY.md#reporting-a-vulnerability) for more information.**

Test Steps:
1. *Include steps to test these changes*

## Changes
- prime-router/docs/onboarding-users/migrating-receivers.md
Added note for `convertDateTimeToReceiverLocalTimezone` and example config
- prime-router/docs/onboarding-users/receivers.md
Added note for example schema
- prime-router/docs/universal-pipeline/translate.md
Added note and linkback to new `migrating-receivers.md` note.

### Testing
- [ ] Tested locally?
- [ ] Ran `./prime test` or `./gradlew testSmoke` against local Docker ReportStream container?
- [ ] (For Changes to /frontend-react/...) Ran `npm run lint:write`?
- [ ] Added tests?

### Process
- [ ] Are there licensing issues with any new dependencies introduced?
- [ ] Includes a summary of what a code reviewer should test/verify?
- [ ] Updated the release notes?
- [ ] Database changes are submitted as a separate PR?
- [ ] DevOps team has been notified if PR requires ops support?

## Linked Issues
- Fixes #16730 

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

## Specific Security-related subjects a reviewer should pay specific attention to
- Does this PR introduce new endpoints?
    - new endpoint A
    - new endpoint B
- Does this PR include changes in authentication and/or authorization of existing endpoints?
- Does this change introduce new dependencies that need vetting?
- Does this change require changes to our infrastructure?
- Does logging contain sensitive data?
- Does this PR include or remove any sensitive information itself?

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?

### CODE REVIEW

File: prime-router/docs/onboarding-users/migrating-receivers.md [hunk 1]
```diff
@@ -68,6 +68,7 @@ In order to migrate existing covid pipeline settings to the UP a few settings ne
 * `schemaName:` Schema name specifies how the RS FHIR bundle should be translated to HL7 if the receiver's format is HL7. 
 If they're receiving HL7 v2 ORU_R01. The schema name can be updated to `azure:/metadata/hl7_mapping/ORU_R01/ORU_R01-base.yml`. 
 If the receiver has any specific receiver transforms the schema name should be updated to point to the schema location.
+* `convertDateTimesToReceiverLocalTime` This setting is no longer able to be used directly. If this setting is present, it should be set to `false`. To enable similar functionality to convert datetimes instead, an enrichment schema can be added under the `enrichmentSchemaNames` list. See also: [Translate#extending-schemas](https://github.com/CDCgov/prime-reportstream/blob/master/prime-router/docs/universal-pipeline/translate.md#extending-schemas)
```
Comment (victor-chaparro) [inline prime-router/docs/onboarding-users/migrating-receivers.md:71]:
```suggestion
* `convertDateTimesToReceiverLocalTime` This setting is no longer able to be used directly. If this setting is present, it should be set to `false`. To enable similar functionality to convert datetimes instead, an enrichment schema can be added under the `enrichmentSchemaNames` list. See also: [Translate#extending-schemas](https://github.com/CDCgov/prime-reportstream/blob/main/prime-router/docs/universal-pipeline/translate.md#extending-schemas)
```

**Review by victor-chaparro — CHANGES_REQUESTED** [review]
Looks good, just had one minor change. We shouldn't reference master anymore. I think I may have sent you that link, so that's my bad.

**Comment by the-andrew** [comment]
> Looks good, just had one minor change. We shouldn't reference master anymore. I think I may have sent you that link, so that's my bad.

Oh, I've been working off main, but my browser bookmark for the top level repo still goes to master. I can update the links.

**Review by oslynn — APPROVED** [review]

File: prime-router/docs/onboarding-users/migrating-receivers.md [hunk 2]
```diff
@@ -120,6 +121,9 @@ After updating the receiver to route messages to the UP it should look like this
         conditionFilter:
             # Accept COVID only
             - "(%resource.code.coding.extension('https://reportstream.cdc.gov/fhir/StructureDefinition/condition-code').value.where(code in ('840539006')).exists())"
+        enrichmentSchemaNames:
+            # Example enrichment schema for US Mountain time zone
+            - "azure:/fhir_transforms/common/datetime-to-local/datetime-to-local-us-mtz.yml"
 ```
```
[the remark this answers is not in the document]
Comment (the-andrew) [reply prime-router/docs/onboarding-users/migrating-receivers.md:133]: Added

**Review by lucero-v — APPROVED** [review]
Looks good! Thank you for testing those settings with me.

**Review by victor-chaparro — APPROVED** [review]

**Pull request merged**